<launch>

  <node name="sender" pkg="chatter_sender" type="chatter_sender" respawn="true" required="false" output="screen" cwd="node" >
    <param name="send_frequency" type="double" value="0.05" />
    <param name="msg_string" type="string" value="ROS Workshop" />
  </node>

</launch>
